src/viking.js

```diff
@@ -1,11 +1,128 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+  this.health = health;
+  this.strength = strength;
+};
+
+// oder soldier = new Soldier(health, strength);??
+
+Soldier.prototype.attack = function () {
+  return this.strength;
+};
+
+Soldier.prototype.receiveDamage = function (damage){
+  this.health -= damage;
+}
 
 // Viking
-function Viking() {}
+function Viking(name, health, strength) {
+  Soldier.call(this, health, strength);
+  this.name =name;
+  this.battleCry= function (){ // is it possible to create this function outside of the Viking function with: Viking.prototype.battleCry? Or would it get overwritten by prototype of Soldier?
+    return "Odin Owns You All!";
+  }
+
+  this.receiveDamage = function (damage) {
+    this.health -= damage;
+    if (this.health > 0) {
+      return this.name + " has received " + damage + " points of damage";
+    } else {
+      return this.name + " has died in act of combat";
+    }
+  };
+  
+
+};
+
+Viking.prototype = Object.create(Soldier.prototype);
+Viking.prototype.constructor = Viking;
+
+
+
 
 // Saxon
-function Saxon() {}
+function Saxon(health, strength) {
+  Soldier.call(this, health, strength);
+  
+  this.receiveDamage = function(damage){
+    this.health -= damage;
+    if (this.health > 0){
+      return "A Saxon has received "+ damage+" points of damage"; 
+    }
+    else {
+      return "A Saxon has died in combat";
+    }
+  };
+};
+
+Saxon.prototype = Object.create(Soldier.prototype);
+Saxon.prototype.constructor = Saxon;
+
+
+
+
+
 
 // War
-function War() {}
+function War() {
+
+  this.vikingArmy = [];
+  this.saxonArmy = [];
+
+  this.addViking = function(viking){
+    this.vikingArmy.push(viking);
+  };
+
+  this.addSaxon = function(saxon){
+    this.saxonArmy.push(saxon);
+  
+  };
+
+
+  this.vikingAttack = function(){
+    let ranNumS = Math.floor(Math.random()*this.saxonArmy.length);
```

Comment:
The code in this method and the code in the method below are very similar. Maybe you could encapsulate it in a function and reused!